stripe/src/main/java/com/stripe/android/Stripe.kt

```diff
@@ -322,6 +322,20 @@ class Stripe internal constructor(
         )
     }
 
+    /**
+     * Check if the requestCode and intent is for [PaymentIntentResult].
+     * The Intent should the retrieved from result from `Activity#onActivityResult(int, int, Intent)}}`
```

Comment:
Is this supposed to say "The `Intent` should be retrieved from the result from ..."?